### PR TITLE
Improve user notifications for new Terms and Conditions

### PR DESCRIPTION
**Pull request by WebSpider**
Add issue creator for new terms and conditions from myskoda

### CODE REVIEW

File: custom_components/myskoda/translations/en.json [hunk 1]
```diff
@@ -213,6 +213,12 @@
             }
         }
     },
+    "issues": {
+        "new_tandc": {
+            "title": "Review new Terms and Conditions for MySkoda",
+	    "description": "Detected login failure due to missing acceptance of Terms and Conditions. Skoda may have issued new Terms and Conditions. Please review them."
```
Comment (dvx76) [inline custom_components/myskoda/translations/en.json:219]:
```suggestion
	    "description": "Detected login failure due to missing acceptance of Terms and Conditions. Skoda may have issued new Terms and Conditions. Please log out and back in of the MySkoda app to accept."
```

**Review by dvx76 — APPROVED** [review]
LGTM. Were you able to test this? Just curious what the issue looks like in HA.

**Comment by WebSpider** [comment]
I tested it, and it works. I will add some screenshots later today.

**Comment by WebSpider** [comment]
Repair notification:
issue1
Example of HA logs:
issue2
Repair text. Clicking "More information" will redirect the user to the login portal of Skoda.
issue3

**Pull request merged**